public/view/main.jsx

```diff
@@ -39,33 +41,38 @@ const SUPPORTED_COUNTRIES = ['br', 'co', 'pa'];
 // ugliness here because AppMain hasn't yet been instantiated.
 var main_instance = null;
 var map_controller = null;
-var rerender = function() {
+var _base_rerender = function() {
   if (main_instance !== null) {
     main_instance.forceUpdate();
   }
 };
-var rerender_and_redraw = function() {
-  rerender();
+var rerender = _.debounce(_base_rerender, 16);  // 16 milliseconds is 60 frames per second ... fast enough
+var rerender_and_redraw = _.debounce(function() {
+  _base_rerender();
   if (map_controller !== null) {
     map_controller.redraw();
   }
-};
+}, 16);
```

Comment:
Yeah, I left a comment on https://github.com/unicef/carrotjuice2/pull/66/commits/9fa85b817ec379012f40ebda2db1dfb8cb747164

I think some of our view rendering (esp. the map) does some non-trivial computation, so this seems like a good thing.